src/crypto/crypto.cpp

```diff
@@ -92,6 +92,13 @@ namespace crypto {
     sc_reduce32(&res);
   }
 
+  void coerce_valid_sec_key_from(const legacy16B_secret_key &legacy16B_mymonero_sec_seed, secret_key &dst__sec_seed) { // cn_fast_hash legacy16B_sec_seed in order to 'pad' it to 256 bits so it can be chopped to ec_scalar
+    static_assert(!epee::has_padding<legacy16B_secret_key>(), "potential hash of padding data");
+    static_assert(!epee::has_padding<secret_key>(), "writing to struct with extra data");
+    cn_pad_by_fast_hash((uint8_t *)&legacy16B_mymonero_sec_seed, sizeof(crypto::legacy16B_secret_key),
+                        (uint8_t *)&dst__sec_seed, sizeof(crypto::secret_key));
```

Comment:
>crypto::

These types don't need to be fully-qualified here (or elsewhere really, other than to keep consistency).

Reply:
Good eye! thanks https://github.com/monero-project/monero/pull/3281/commits/fa67a4bdf507c1c3781e4609176ac5f149775d61#diff-1fbbac059f595e941719bce40a95484aR98